src/SequenceMsg.chpl

```diff
@@ -55,7 +56,38 @@ module SequenceMsg {
         repMsg = "created " + st.attrib(rname);
         smLogger.debug(getModuleName(),getRoutineName(),getLineNumber(),repMsg);
         return new MsgTuple(repMsg, MsgType.NORMAL);
-    }            
+    }
+
+    proc bigIntArangeMsg(cmd: string, msgArgs: borrowed MessageArgs, st: borrowed SymTab): MsgTuple throws {
+        var repMsg: string; // response message
+        var start = msgArgs.get("start").getBigIntValue();
+        var stop = msgArgs.get("stop").getBigIntValue();
+        var stride = msgArgs.get("stride").getBigIntValue();
+        // compute length
+        var len = (stop - start + stride - 1) / stride;
+        // overMemLimit(8*len);

```

Comment:
Just noting here that we really need to determine a method for computing memory allocation for `BigInt`. I know that is not the easiest thing to do with how it functions, but we should definitely put in an issue to look into this.

Reply:
yeah for sure, this is something I would lean on one of the chapel guys for @ronawho and @bmcdonald3 since it required knowing how many bits are in the type `bigint` which may vary unlike other types

Reply:
It seems that there is a `sizeInBase` method (https://chapel-lang.org/docs/modules/standard/BigInteger.html#BigInteger.bigint.sizeInBase) for `bigint` that could be used to compute this, but I think it would be pretty expensive to run that over every element in the pdarray every time the size is queryed... Maybe we could calculate it once upon creation and cache that value? I am not enough of a `bigint` expert to know exactly what that size calculation would look like.

Reply:
I created an issue to track this (#1977)